tests/test_create_library.py

```diff
@@ -0,0 +1,86 @@
+from sqlalchemy_redshift import commands
+from rs_sqla_test_utils.utils import clean, compile_query
+
+ACCESS_KEY_ID = 'IO1IWSZL5YRFM3BEW256'
+SECRET_ACCESS_KEY = 'A1Crw8=nJwEq+9SCgnwpYbqVSCnfB0cakn=lx4M1'
```

Comment:
Did you intend for these keys to get committed in this PR?

Reply:
I see now these are copied from another test as introduced in https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/pull/21

I hadn't been aware that these creds were here and don't quite understand the history of how they got there. Perhaps @graingert remembers more? Was this intended?

Reply:
I'm pretty sure they're inert keys to test the regex

Reply:
Might be worth making it more obvious that they are example creds

Reply:
Oh yeah sorry about that. Will fix it.

Reply:
Okay, added a comment.